hourly_logger_rotator.gemspec

```diff
@@ -17,6 +17,7 @@ Gem::Specification.new do |spec|
   spec.files         = `git ls-files -z`.split("\x0").reject { |f| f.match(%r{^spec/}) }
   spec.require_paths = ["lib"]
 
+  spec.add_dependency             "activesupport"
```

Comment:
go ahead?

Reply:
Please remove it.

Reply:
And by the way, you didn't even require the proper part in the gem code.

Reply:
Say, you'd prefer to use following syntax
`HourlyLoggerRotator.logs_lifetime=(604800)`
# instead of 
`HourlyLoggerRotator.logs_lifetime=(1.week)`
and it is very likely that this gem will be used inside rails projects

Reply:
The gem itself does not need this, rails application will still be able to use 1.week, etc